app/src/sharedTest/java/org/oppia/app/splash/SplashActivityTest.kt

```diff
@@ -0,0 +1,42 @@
+package org.oppia.app.splash
+
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.Intents.intended
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.home.HomeActivity
+
+/** Tests for [SplashActivity].
+ * https://jabknowsnothing.wordpress.com/2015/11/05/activitytestrule-espressos-test-lifecycle */
+@RunWith(AndroidJUnit4::class)
+class SplashActivityTest {
+
+  // The initialTouchMode enables the activity to be launched in touch mode.The launchActivity is disabled to launch Activity explicitly within each test case
```

Comment:
Please wrap this according to the line wrapping:

```kotlin
  // The initialTouchMode enables the activity to be launched in touch mode. The launchActivity is
  // disabled to launch Activity explicitly within each test case.
```

Reply:
resolved comment issue